src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -289,10 +299,51 @@ public static int getFuelValue(ItemStack itemStack)
         return fuelValue;
     }
 
+    /**
+     * Links a set of parts to an {@link IPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IPartFactory factory, ResourceLocation... names)
+    {
+        MultipartRegistry.registerPartFactory(factory, names);
+    }
+
+    /**
+     * Links a set of parts to an {@link IAdvancedPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IAdvancedPartFactory factory, ResourceLocation... names)
+    {
+        MultipartRegistry.registerPartFactory(factory, names);
+    }
+
+    /**
+     * Registers a part along with an identifier. A default part factory is automatically created.
+     */
+    public static void registerPart(Class<? extends IMultipart> clazz, ResourceLocation name)
+    {
+        MultipartRegistry.registerPart(clazz, name);
+    }
+
+    /**
+     * Registers an {@link IPartConverter}.
+     */
+    public static void registerPartConverter(IPartConverter converter)
+    {
+        MultipartRegistry.registerPartConverter(converter);
+    }
+
+    /**
+     * Registers an {@link IReversePartConverter}.
+     */
+    public static void registerReversePartConverter(IReversePartConverter converter)
```

Comment:
If there was a way to make a registry that doesn't have keys, sure, but these things don't really require an ID and I don't think users should be forced to add one if it's never going to be used for anything.